templates/incl-adventure-tabs.html

```diff
@@ -1,81 +1,42 @@
       {# TABS #}
       <div id="adventures">
         <div class="flex flex-row overflow-x-scroll overflow-y-hidden mb-2 px-2 pt-2">
-            <div class="tab z-10 tab-selected whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="level">
-                {{ _('level_title') }} {{ level_nr }}
-            </div>
-            {% for adventure in teacher_adventures %}
-                {% if adventure.level == level_nr %}
-                    <div class="tab teacher_tab z-10 whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="{{adventure.name}}">
-                        {{ adventure.name }}
-                    </div>
-                {% endif %}
-            {% endfor %}
-            {% for assignment in adventures %}
-              {% if assignment.short_name != 'level' and ('adventures' not in customizations or customizations['adventures'][assignment.short_name] and level|int in customizations['adventures'][assignment.short_name]) %}
-                <div class="tab {% if assignment.short_name == "money" %}money_tab{% endif %} z-10 whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="{{assignment.short_name}}">
-                    {{ assignment.name }}
+            {% for adventure in adventures %}
+              {% if 'adventures' not in customizations or (customizations['adventures'][adventure.short_name] and level|int in customizations['adventures'][adventure.short_name]) %}
+                <div class="tab {% if loop.index == 1 %}tab-selected{% endif %} z-10 whitespace-nowrap flex items-center justify-center" tabindex="0" data-tab="{{adventure.short_name}}">
```

Comment:
Is that `loop.index` to do something special for the intro tab? Maybe add a comment?

Reply:
This is independent of the intro tab, we simply want to set the first tab as the selected one by default. When there is a loaded program we overwrite the `tab-selected` with the loaded adventure.